### PR TITLE
Translation set up

### PR DESCRIPTION
**Pull request by davidwillprice**

### CODE REVIEW

File: src/components/layout/yearInReview/2021/animatedVideos.tsx [hunk 1]
```diff
@@ -22,16 +24,28 @@ const AnimatedVideos: React.FC = () => {
...
-            <CustomLink href="https://expandedcircle.org/">
-              Expanded Circle Collective
-            </CustomLink>{' '}
-            in order to produce two animated introductory videos for Activist
-            Hub and Vegan Bootcamp! These videos are fantastic resources for
-            viewers to quickly get familiar with our work. We look forward to
-            collaborating with them in the new year.
+            <FormattedMessage
+              id="page.year-in-review.2021.section.videos.paragraph"
+              defaultMessage="We've partnered with <no-localization>{link}</no-localization> in order to produce two animated introductory videos for <no-localization>Activist Hub</no-localization> and <no-localization>Vegan Bootcamp</no-localization>! These videos are fantastic resources for viewers to quickly get familiar with our work. We look forward to collaborating with them in the new year."
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/animatedVideos.tsx]: To make the link text changeable by translators, please apply these changes
```
defaultMessage="We've partnered with <link><no-localization>Expanded Circle Collective</no-localization></link> in order to produce two animated introductory videos for <no-localization>Activist Hub</no-localization> and <no-localization>Vegan Bootcamp</no-localization>! These videos are fantastic resources for viewers to quickly get familiar with our work. We look forward to collaborating with them in the new year."

values={{                link: (chunks) => (                  <CustomLink href="https://expandedcircle.org/">                    {chunks}                  </CustomLink>                ),              }}
```

File: src/components/layout/yearInReview/2021/byTheNumbers.tsx [hunk 2]
```diff
@@ -44,56 +54,98 @@ const ByTheNumbers: React.FC = () => {
...
-                <a>
-                  <b>veganactivism.org</b>
-                </a>
-              </Link>
+              <FormattedMessage
+                id="page.year-in-review.2021.section.by-the-numbers.statistic.2"
+                defaultMessage="Clicks directing activists to orgs {link}"
+                values={{
+                  link: (
+                    <Link href="https://veganactivism.org">
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/byTheNumbers.tsx:89]: Please move the link label wrapped in a no-localization tag to the default message

File: src/components/layout/yearInReview/2021/byTheNumbers.tsx [hunk 3]
```diff
@@ -44,56 +54,98 @@ const ByTheNumbers: React.FC = () => {
...
               number={
                 <AnimatedNumber number={500000} className="text-green" approx />
               }
             >
-              Unique page visits for <b>Vegan Bootcamp</b>
+              <FormattedMessage
+                id="page.year-in-review.2021.section.by-the-numbers.statistic.3"
+                defaultMessage="Unique page visits for {project}"
+                values={{
+                  project: <b>Vegan Bootcamp</b>,
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/byTheNumbers.tsx]: Same as above. Text should always be inside the defaultMessage, to keep it configurable by translators.

File: src/components/layout/yearInReview/2021/communityGrowth.tsx [hunk 4]
```diff
@@ -53,12 +71,22 @@ const growthPoints: {
...
 
 const CommunityGrowth: React.FC = () => {
+  const intl = useIntl();
   return (
     <>
       <SectionContainer
         header={
           <SectionHeader
-            header={['Encouraging', 'community growth']}
+            header={[
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/communityGrowth.tsx]: Please leave this out for now, I'll look for a solution to translate this in one message.

File: src/components/layout/yearInReview/2021/designsForVeganOrgs.tsx [hunk 5]
```diff
@@ -7,19 +9,34 @@ import SectionContainer from '../sectionContainer';
...
 
 const DesignsForVeganOrgs: React.FC = () => {
+  const intl = useIntl();
   return (
     <>
       <SectionContainer
         color="grey-background"
         header={
           <SectionHeader
             className="text-grey-dark"
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/designsForVeganOrgs.tsx]: Same here.

File: src/components/layout/yearInReview/2021/featuredBlogPosts.tsx [hunk 6]
```diff
@@ -22,19 +23,28 @@ export interface FeaturedBlogPostsProps {
...
 }) => {
+  const intl = useIntl();
   return (
     <>
       <SectionContainer
         header={
           <SectionHeader
             className="text-grey"
-            header={['Featured', 'blog posts']}
+            header={[
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/featuredBlogPosts.tsx]: And here, everywhere the section header is used this way.

File: src/components/layout/yearInReview/2021/partnerships.tsx [hunk 7]
```diff
@@ -35,90 +37,119 @@ const Partnership: React.FC<PartnershipProps> = ({
...
               <CustomLink href="https://veganoutreach.org">
                 <span className="block md:inline">
                   <b>Vegan Outreach</b>
                 </span>
               </CustomLink>
               <span className="hidden md:inline">&nbsp;&mdash;&nbsp;</span>
-              We partnered with Vegan Outreach&apos;s Vegan Mentorship Program
-              in order to provide completely free mentorship access to all Vegan
-              Bootcamp and 3Movies.org participants.
+              <FormattedMessage
```
Comment (MatsThieme) [inline src/components/layout/yearInReview/2021/partnerships.tsx:145]: Please add a FormattedMessage for the above Vegan Outreach link.

**Review by MatsThieme — CHANGES_REQUESTED** [review]
To make sure everything is formatted correctly please run pnpm format.

File: src/components/layout/yearInReview/2021/byTheNumbers.tsx (hunk 3 above)
Comment (davidwillprice) [reply src/components/layout/yearInReview/2021/byTheNumbers.tsx]: I am a bit confused by this. As `Vegan Bootcamp` is an organisation, don't we want the translators to ignore it anyway? IIRC I've hopefully only included text inside the `values` if we don't want it translated anyway.

File: src/components/layout/yearInReview/2021/partnerships.tsx (hunk 7 above)
Comment (davidwillprice) [reply src/components/layout/yearInReview/2021/partnerships.tsx:145]: Isn't Vegan Outreach an organisation so we wouldn't want it translated? Or would you want a `defaultMessage` that's just `defaultMessage="<no-localization>Vegan Outreach</no-localization>"` to better facilate future changes or something?

File: src/components/layout/yearInReview/2021/byTheNumbers.tsx (hunk 3 above)
Comment (MatsThieme) [reply src/components/layout/yearInReview/2021/byTheNumbers.tsx]: Let's say Vegan Bootcamp is renamed (hypothetically, highly unlikely but not impossible), we'd have to update this inside the code. Instead I'd prefer all text managed in one source which would be the json files/weblate.

File: src/components/layout/yearInReview/2021/partnerships.tsx (hunk 7 above)
Comment (MatsThieme) [reply src/components/layout/yearInReview/2021/partnerships.tsx:145]: The latter, text changes should be done easily by anyone without having to dig through the code.

File: src/components/layout/yearInReview/2021/intro.tsx [hunk 8]
```diff
@@ -20,49 +23,92 @@ const Intro: React.FC = () => {
...
+            defaultMessage: 'We launched',
+          })}
+          headerBold={intl.formatMessage({
+            id: 'page.year-in-review.2021.section.we-grew.projects.heading.1',
+            defaultMessage: '8 new projects',
+          })}
+          headerEnd={intl.formatMessage({
+            id: 'page.year-in-review.2021.section.we-grew.projects.heading.2',
+            defaultMessage: 'for the movement',
+          })}
```
Comment (davidwillprice) [inline src/components/layout/yearInReview/2021/intro.tsx]: Should I remove the translations for `headerStart`, `headerBold`, and `headerEnd` too @MatsThieme?

Comment (MatsThieme) [reply src/components/layout/yearInReview/2021/intro.tsx]: Yes!

Comment (davidwillprice) [reply src/components/layout/yearInReview/2021/intro.tsx]: No problem, deleted them in my last commit

**Review by MatsThieme — APPROVED** [review]

**Pull request merged**